backend/api/v2beta1/pipeline.proto

```diff
@@ -233,7 +233,7 @@ message ListPipelinesRequest {
   string sort_by = 4;
 
   // A url-encoded, JSON-serialized filter protocol buffer (see
-  // [filter.proto](https://github.com/kubeflow/pipelines/blob/master/backend/api/filter.proto)).
+  // [filter.proto](https://github.com/kubeflow/pipelines/blob/master/backend/api/v1beta1/filter.proto)).
```

Comment:
This looks not correct. We have [filter.proto](https://github.com/kubeflow/pipelines/blob/master/backend/api/v2beta1/filter.proto) in v2beta1. Why reference to v1beta1?